Repositories/ExperiencesRepository.cs

```diff
@@ -0,0 +1,66 @@
+using rexfinder_api.Migrations;
+using rexfinder_api.Models;
+using rexfinder_api.Repositories;
+
+public class ExperiencesRepository : IExperiencesRepository 
+{
+    private readonly MyPlacesDbContext _context;
+
+    public ExperiencesRepository(MyPlacesDbContext context)
+    {
+        _context = context;
+    }
+
+    public Experiences CreateExperience(Experiences newExperience)
+    {
+        _context.Experiences.Add(newExperience);
+        _context.SaveChanges();
+        return newExperience;
+    }
+
+    public IEnumerable<Experiences> GetAllExperiencesByUserId(int userId)
+    {
+        var experienceList = _context.Experiences.Where(e => e.UserId == userId).ToList();
+        return experienceList;
+    }
+
+    public Experiences GetExperiencesById(int experienceId)
+    {
+        return _context.Experiences.SingleOrDefault(e => e.ExperienceId == experienceId);
+    }
+
+    public Experiences GetExperiencesByUserIdGoogleId(int userId, string googlePlaceId)
+    {
+        var experienceList = this.GetAllExperiencesByUserId(userId);
+
+        var foundPlace = experienceList.FirstOrDefault(e => e.FirstGooglePlaceId == googlePlaceId);
+
+        return foundPlace;
+    }
```

Comment:
Same -- `foundExperience`

Reply:
eh it would have still worked since it used the same name but fixed